test/functionsTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
Functions not currently tested

Reply:
I've completed success test, will begin working on failure and malform test.

Reply:
2 more to go